node/src/Pos/Wallet/Web/ClientTypes/Functions.hs

```diff
@@ -130,10 +130,10 @@ mkCTx
     -> TxHistoryEntry     -- ^ Tx history entry
     -> CTxMeta            -- ^ Transaction metadata
     -> CPtxCondition      -- ^ State of resubmission
-    -> Set (CId Addr)     -- ^ Addresses of wallet
+    -> (CId Addr -> Bool) -- ^ Whether addresses belongs to the wallet
     -> Either Text CTx
-mkCTx diff THEntry {..} meta pc wAddrsSet = do
-    let isOurTxAddress = flip S.member wAddrsSet . addressToCId . txOutAddress
+mkCTx diff THEntry {..} meta pc addrBelongsToWallet = do
+    let isOurTxAddress = addrBelongsToWallet . addressToCId . txOutAddress
```

Comment:
We are safe, shipped as part of [this commit](https://github.com/input-output-hk/cardano-sl/commit/baf8c4c39533949626c2972b90d9a4886621258a)